sp800-63b/errata.md

```diff
@@ -26,4 +26,4 @@ This table contains changes that have been incorporated into Special Publication
 ||Editorial|Emphasized use of key unlocked by additional factor to sign nonce|§5.1.9.1|
 ||Editorial|Provided examples of risk-based behavior observations|§5.2.2|
 ||Editorial|Updated URL for reference [Blacklists]|§11.1|
-||Editorial|Remove redundant phrase|§5.2.3|
\ No newline at end of file
+||Editorial|Remove redundant phrase|§5.2.3|
```

Comment:
Wow, newline at the end was the problem. No wonder I didn't see it.

Reply:
It took a bunch of trial and error on my side to find that, but now we know! Everything else looks good by my read.